contents/manual/feature-flags.mdx

```diff
@@ -436,4 +436,7 @@ Want to know more about what's possible with Feature Flags in PostHog? Try these
 - [Running experiments on new users](/tutorials/new-user-experiments)
 - [How to run Experiments without feature flags](/tutorials/experiments)
 
+Using a library other than JavaScript for your feature flag implementation? Check out these other libraries for more details.
```

Comment:
```suggestion
Using a library other than JavaScript for your feature flag implementation? Check out [these other libraries](/docs/integrate/libraries) for more details.
```

a list makes sense if we're listing all libraries 😋

Reply:
yeah something like a matrix that shows all libraries and what functionalities of feature flags they support would be nice. Since we're still updating some of our libraries though, we can wait on this a little